### PR TITLE
schutzbot: Always grab versions from terraform repo

### PR DESCRIPTION
**Pull request by Gundersanne**
Exit 1 in case this fails. We always want to test IB against the current
version in stage.

### CODE REVIEW

File: schutzbot/deploy.sh [hunk 1]
```diff
@@ -22,57 +22,25 @@ fi
...
-          CURL_URL="https://api.github.com/repos/osbuild/image-builder-terraform/contents/terraform.tfvars.json?ref=$IB_TERRAFORM_BRANCH"
-          TERRAFORM_VARS=$(curl -H "Authorization: token $GITHUB_TOKEN" -H "Accept: application/vnd.github.v3.raw" -L "$CURL_URL")
-          OSBUILD_COMMIT=$(echo "$TERRAFORM_VARS" | jq -r .osbuild_commit)
-          OSBUILD_COMPOSER_COMMIT=$(echo "$TERRAFORM_VARS" | jq -r .composer_commit)
-     fi
-fi
-
+# Retrieve composer&osbuild version from current stage deployment
+pushd "$(mktemp -d)"
+chmod 400 "$TERRAFORM_REPO_DEPLOY_KEY"
```
Comment (jrusz) [inline schutzbot/deploy.sh:27]: Maybe it needs `sudo` here? Similar to line 17 when making the registration script executable?

Comment (Gundersanne) [reply schutzbot/deploy.sh]: I think the entire thing mihgt need to be sudo, if it's 400 for root then it won't be readable by the user executing the pipeline.

Comment (Gundersanne) [reply schutzbot/deploy.sh]: eh i'll just keep iterating until it's green here :D

**Review by teg — APPROVED** [review]
Makes sense. As a follow up is it possible to actually run tests against stage itself though?

**Comment by Gundersanne** [comment]
> Makes sense. As a follow up is it possible to actually run tests against stage itself though?

Yea that's a good idea, but let's unbreak CI first :D

**Pull request merged**